README.md

```diff
@@ -1,39 +1,15 @@
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-**Table of Contents**  *generated with [DocToc](https://github.com/thlorenz/doctoc)*
+<img src="https://www.kubeflow.org/images/logo.svg" width="100">
```

Comment:
I think we could make the image in the center like 

```html
<h1 align="center">
	<br>
	<img width="400" src="docs/images/logo.jpeg" alt="cyclone">
	<br>
</h1>
```

Reply:
I tried that but it does not look good :-(

<img width="975" alt="screen shot 2018-07-20 at 8 31 13 pm" src="https://user-images.githubusercontent.com/2220322/43009722-09d1adae-8c5c-11e8-8569-fd8a425a5213.png">